README.md

```diff
@@ -33,23 +33,15 @@ providers out of the box, which are both under active development:
 
 ## Demos
 
-The demo at https://h5web.panosc.eu/ shows the content of an HDF5 file converted
-to JSON with [hdf5-json](https://github.com/HDFGroup/hdf5-json) and hosted on a
-remote static server.
-
-At https://h5web.panosc.eu/mock, you can view a set of mock data generated
-entirely on the front-end. This demo and its provider, `MockProvider` are used
-for end-to-end testing purposes.
-
 ### [Jupyter Lab HDF5 extension](https://github.com/jupyterlab/jupyterlab-hdf5)
```

Comment:
I think the title should be reworded as one may think it is a JLab extension and not a stand-alone app.

Reply:
I've added a little intro, does it clarify things?

Reply:
It does :+1: